File: fibonacci.cpp
#include <iostream>
#include <vector>
using namespace std;

void fibonacci(int num_terms) {
    vector<int> fib_sequence(num_terms, 0);
    fib_sequence[1] = 1;

    for (int i = 2; i < num_terms; ++i) {
        fib_sequence[i] = fib_sequence[i - 1] + fib_sequence[i - 2];
    }

    cout << "Fibonacci sequence of " << num_terms << " terms:" << endl;
    for (int num : fib_sequence) {
        cout << num << " ";
    }
    cout << endl;
}

int main() {
    int num_terms = 10;  // Change the number of terms in the sequence as needed
    fibonacci(num_terms);

    return 0;
}

File: fibonacci.m
% Fibonacci Sequence in MATLAB

function fib_sequence = fibonacci(num_terms)
    % Function to generate the Fibonacci sequence up to a specified number of terms
    % Input: num_terms - Number of terms in the Fibonacci sequence
    % Output: fib_sequence - Array containing the Fibonacci sequence
    
    fib_sequence = zeros(1, num_terms);
    fib_sequence(1) = 0;
    fib_sequence(2) = 1;
    
    for i = 3:num_terms
        fib_sequence(i) = fib_sequence(i - 1) + fib_sequence(i - 2);
    end
    
    fprintf('Fibonacci sequence of %d terms:\n', num_terms);
    disp(fib_sequence);
end